11-payment-encoding.md

```diff
@@ -277,6 +277,10 @@ The field is big-endian.  The least-significant bit is numbered 0,
 which is _even_, and the next most significant bit is numbered 1,
 which is _odd_.
 
+| Bits | Name        |Description            | Link                                           |
+|------|-------------|-----------------------|------------------------------------------------|
+| 0/1  | `basic_mpp` | Payee understands mpp.| [BOLT #4](04-onion-routing.md#basic-multi-part-payments) |
```

Comment:
What would be the point of using the even bit?

For odd bits, it seems to me to mean, "payer *may* use `basic_mpp`".

For even bits, what is it mean? "Payer *must* use `basic_mpp`"?

Reply:
We always assign in pairs.  In future, it's theoretically possible you could set this if no single channel had sufficient capacity, I guess, but it's a bit weird!

Reply:
We may also want to add an additional section here w.r.t path finding advisory using routing hints. If the amount of the invoice is greater than available inbound bandwidth of any of their channels, then they can "guide" the sender with regular or "special" routing hints to increase the likelihood of success.